lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +263,141 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        text_elements = article_soup.find("div", class_="page-main__text").find_all("p")
+        self.article.text = "\n".join([p.get_text(strip=True) for p in text_elements])
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        author_elem = article_soup.find_all('a', class_='page-main__publish-author global-link')[0]
```

Comment:
is there a better way to find author? sequence of classes is fragile. (here and further)

Reply:
done